### ISSUE TITLE
/lib64/libstdc++.so.6: version `GLIBCXX_3.4.30' not found

### ISSUE THREAD

**Issue by Reinitialized** [JohnnyMorganz/luau-lsp#427]
# Overview
- I am attempting to utilize this extension in Visual Studio Code using the Remote - SSH extension, I am running into the following error message, resulting in the extension crashing and exiting: `/lib64/libstdc++.so.6: version `GLIBCXX_3.4.30' not found`
- I am running `AlmaLinux 9.2 (Turquoise Kodkod)` with the "Development Tools" group packages installed in attempts to resolve this issue.
- I have extensively done Google-fu along with utilizing BingGPT/ChatGPT in attempts to verify this is not a end-user issue, but no solutions can be found with lots of various user complaints for independent projects running into a similar issue.
- Solutions I am able to find seem to point to the project itself needing to adapt their compiling code to support a relatively recent change that occurred in what appears to be 2020.

**Comment by JohnnyMorganz** [comment]
Apologies for the delay in responding, I was away.

This error typically means you have an old version C++ toolchain(? - not sure exact terminology). It often happens in the outdated Ubuntu 18.04 version, and I guess AlmaLinux has the same issue. Our builds run on Ubuntu 22.04, which uses a newer glibc.

There are two ways you can resolve this:
1) Build the `server` binary manually as mentioned in the README, and then replace the binary in the extension using this one (or alternatively package the extension yourself in a new vsix)
2) Update `libstdc++`. I *think* you can do this by doing `sudo apt-get install libstdc++6`, but please verify this yourself first.

**Comment by Reinitialized** [comment]
No worries Johnny, we all need breaks :)

Yeah, I learned about that trying to get this all up and going and forgot to update this issue, whoopsies! Perks of using your own hosted Git solution ...

I already tried the manual build route and ran into a separate issue, most likely older libraries again, I don't remember as I didn't document the process. Unfortunately, I tried updating libstdc++ as well through the package manager, but nothing newer seemed available, leaving trying to manually compile a newer version of gcc++ which is a long long process even with server hardware and tossing some 32 cores at it :)

For now, I went back to working out of Studio, but will probably approach this again some time down the road when my ADHD needs something random to lock onto. If so, I'll be sure to document my resolution and update accordingly for those who struggle with these newer library requirements as well.
